setup.py

```diff
@@ -587,6 +587,7 @@ def get_ddup_ext():
         "envier",
         "opentelemetry-api>=1",
         "setuptools; python_version>='3.12'",
+        "sqlparse>=0.4.4",
```

Comment:
This is the current release. Do we want the minimum version to be different in order to not cause conflicts with users pinned to an older version?

Reply:
Yeah, good idea, seems to work with 0.2.2 onwards so I changed the dependency to it.